engine.py

```diff
@@ -225,14 +258,18 @@ def transcribe(self, path):
         response = self._client.recognize(config=self._config, audio=audio)
```

Comment:
same warning about arguments for `RecognitionAudio`. If it is a version thing, we should lock the version of the package

Reply:
I locked the version. the warnings are a bit unclear to me as the docs confirm the datatype that we input.